swarm/src/behaviour.rs

```diff
@@ -549,15 +549,21 @@ pub struct ListenerClosed<'a> {
 }
 
 /// [`FromSwarm`] variant that informs the behaviour
-/// that we have discovered a new external address for us.
+/// that we have discovered a new candidate for an external address for us.
 #[derive(Clone, Copy)]
-pub struct NewExternalAddr<'a> {
+pub struct NewExternalAddrCandidate<'a> {
     pub addr: &'a Multiaddr,
 }
 
 /// [`FromSwarm`] variant that informs the behaviour that an external address was removed.
 #[derive(Clone, Copy)]
-pub struct ExpiredExternalAddr<'a> {
+pub struct ExternalAddrConfirmed<'a> {
+    pub addr: &'a Multiaddr,
+}
```

Comment:
Are these docs correct? They seem to belong to the type below?
I ended up in the pr because docs don't seem to relate to the type name